src/pages/MyProfile/MyProfile.js

```diff
@@ -61,7 +77,9 @@ const MyProfile = ({ history: { push } }) => {
         </small>}
       </label>
 
-      <label styleName='field'>
+      {/* TODO: Disuss email display options with C & Z. - Do we want to display the email here at all ? */}
```

Comment:
is this captured in pivotal?

Reply:
So, I brought it up in stand-up yesterday, but currently the decision is to not focus my profile as it will  won't be included in the first iteration.  I will check and make a sure there is a backlog item to earmark this discussion though.